packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -117,25 +130,275 @@ export default class VTable {
      * @param format Table format to apply
      */
     applyFormat(format: Partial<TableFormat>) {
-        if (!format || !this.table) {
+        if (!this.table) {
             return;
         }
+        if (format) {
+            this.formatInfo = format;

```

Comment:
And since format is Partial type here, we need to make sure all properties have meaningful value.

you can do 
```ts
this.formatInfo = {
   ...defaultFormat, // Make sure everything has value
   ...(this.formatInfo || {}), // Apply existing format if any
   ...(format || {})  // Then overwrite with the passed-in value
};
```